templates/conversational-feedback/README.md

```diff
@@ -0,0 +1,86 @@
+# Chat Feedback Template
+
+This template captures implicit feedback from human behavior in a simple chat bot. It instructs an LLM to reference a user's responses within a conversation to evaluate the chat bot's previous replies.
```

Comment:
Nice. 

1/ IIUC, this is performing chat evaluations w/o explicit user-feedback, which is very useful. We might create a top-level shortened summary that just states this clearly. It's the first eval template, so very cool to have.

2/ We might explicitly mention that your chat app should be implemented (or called) in `chain.py` and call out specifically where as a placeholder. AFAICT, any chat runnable can simply append:
```
    .with_config(
        run_name="ChatBot",
        callbacks=[
            EvaluatorCallbackHandler(
                evaluators=[
                    ResponseEffectivenessEvaluator(evaluate_response_effectiveness)
                ]
            )
        ],
```

3/ Where to go fetch the evals in LangSmith? May be nice to show screenshot.

Reply:
Cool cool. For (1), are you saying on top of the README that's here? (is it too roundabout)?

For (2) - yes albeit it needs a 'last_run_id' to be passed around so that the feedback can be assigned to the previous response trace. If we didn't care about the exact credit assignment, or if we had a better way of tracking conversations, this would be easier/better

For (3) def. I'll do that.